lib/components/normal-components/Board.ts

```diff
@@ -146,11 +146,31 @@ export class Board extends Group<typeof boardProps> {
   }
 
   doInitialPcbDesignRuleChecks() {
+    if (this.root?.pcbDisabled) return
+    if (this.getInheritedProperty("routingDisabled")) return
+    this.updatePcbDesignRuleChecks()
```

Comment:
don't call lifecycle methods directly, you may need a hidden function e.g. `_doDrcChecks()`